rust/processor/src/db/postgres/models/events_models/events.rs

```diff
@@ -11,6 +11,7 @@ use aptos_protos::transaction::v1::Event as EventPB;
 use field_count::FieldCount;
 use serde::{Deserialize, Serialize};
 
+// Question: Do we have to add a replay_protection_nonce here?
```

Comment:
up to you. it's useful for users but if you want it for debugging we could add it.